packages/node/src/types/config.ts

```diff
@@ -181,6 +219,13 @@ export const LocalEvaluationDefaults: LocalEvaluationConfig = {
   bootstrap: {},
   flagConfigPollingIntervalMillis: 30000,
   httpAgent: null,
+  getFlagConfigUpdateWithStream: false,
+  streamServerUrl: 'https://stream.lab.amplitude.com',
+  streamConnTimeoutMillis: 1000,
+  streamFlagConnTimeoutMillis: 1000,
+  streamFlagTryAttempts: 2,
+  streamFlagTryDelayMillis: 1000,
+  retryStreamFlagDelayMillis: 15000,
```

Comment:
Do we want to adjust these defaults? Such as reducing `streamFlagTryAttempts` to 1 for quicker start ups if stream is unavailable and increasing `retryStreamFlagDelayMillis` for longer interval between checks.